pkg/packetbrokeragent/grpc_gspba.go

```diff
@@ -157,16 +163,44 @@ func (s *gsPbaServer) UpdateGateway(ctx context.Context, req *ttnpb.UpdatePacket
 		}
 	}
 
+	var adminContact, techContact *packetbroker.ContactInfo
 	if ttnpb.HasAnyField(req.FieldMask.GetPaths(), "contact_info") {
-		adminContact, techContact := toPBContactInfo(req.Gateway.GetContactInfo())
-		updateReq.AdministrativeContact = &packetbroker.ContactInfoValue{
-			Value: adminContact,
+		adminContact, techContact = toPBContactInfo(req.Gateway.GetContactInfo()) // nolint: staticcheck
+	}
```

Comment:
Just verifying that it is essential that we carry `contact_info` from IS to GS to PBA?

Reply:
It isn't essential as the field has been deprecated.

There was a [previous discussion](https://github.com/TheThingsNetwork/lorawan-stack/pull/6430#discussion_r1291305443) in this PR which the topic wasn't exactly your question but it does bring up the reason as to why pass the `contact_info` around. The main argument is that while the field is deprecated the expected behavior was to have it propagated between these components, so it makes sense to have it included until we remove it completely.

Reply:
So I was wrong, my bad. I thought that we still need this information, but that's not the case: https://github.com/TheThingsNetwork/lorawan-stack/pull/4996 already migrated the contacts on a best effort basis, and I should've checked that PR first. Notifications completely skip `contact_info` today, so it's really legacy data that is not used internally anymore (with a small exception for users).

We can already drop the reference to `contact_info` in this PR, and in a subsequent tech debt PR start removing the storage of the fields. We can keep the entity `contact_info` as a readonly view of the `technical_contact` and `administrative_contact` fields, while disallowing updates.

Reply:
>So I was wrong, my bad. I thought that we still need this information, but that's not the case: https://github.com/TheThingsNetwork/lorawan-stack/pull/4996 already migrated the contacts on a best effort basis, and I should've checked that PR first.

Same, specially since the PR mentioned was linked on the discussion issue and I should've caught that. Regardless, will work on removing the `contact_info` in this PR  and will create the issue for the tech_debt suggested by Adrian